src/main/java/net/phptravels/gui/login/HomePage.java

```diff
@@ -17,84 +15,216 @@
-    private ExtendedWebElement toCountryTextbox;
+    @FindBy(xpath = "//form[@id = 'tours-search']//a[@class = 'dropdown-toggle dropdown-btn travellers waves-effect']")
+    private ExtendedWebElement toursTravellersDropdown;
     
-    @FindBy(xpath ="//div[text()= '28-04-2022']")
+    @FindBy(xpath = "//form[@id = 'tours-search']//button[@id = 'submit']")
+    private ExtendedWebElement toursSearchButton;
+    
+    @FindBy(xpath = "//div//button[@id = 'visa-tab']")
+    private ExtendedWebElement visaButton;
+    
+    @FindBy(xpath = "//div[@id='visa']//span[@data-select2-id = 5]")
+    private ExtendedWebElement visaFromCountryComboBox;
+    
+    @FindBy(xpath = "//div[@id='visa']//span[@data-select2-id = 7]")
+    private ExtendedWebElement visaToCountryComboBox;
+    
+    @FindBy(xpath ="//div[@id='visa']//input[@id = 'date']")
     private ExtendedWebElement visaDate;
     
+    @FindBy(xpath = "//div[@id='visa']//button[@type = 'submit']")
+    private ExtendedWebElement visaSubmitButton;
+     
+    @FindBy(xpath = "//input[@class='select2-search__field']")
+    private ExtendedWebElement universalInputField;
+    
+    @FindBy(xpath = "//span[@class='select2-results']//li")
+    private ExtendedWebElement universalSearchResult;
+    
     public HomePage(WebDriver driver) {
         super(driver);
     }
 
     public Topbar topbar() {
         return topbar;
     }
+    
+    
+    public void hotelSearchForm(String cityName, String checkin, String checkout, String rooms, String adults, String childs) {
+    	
+    	hotelDestinationCombobox.click();
+    	universalInputField.type(cityName);
+    	universalSearchResult.click();
+    	
+    	hotelCheckinDate.click();
+    	hotelCheckinDate.type(checkin);
+    	hotelCheckoutDate.click();
+    	hotelCheckoutDate.type(checkout);
+    	
+    	hotelTravellersCombobox.click();
+    	hotelRoomsInput.type(rooms);
+    	hotelAdultsInput.type(adults);
+    	hotelCheckinDate.type(childs); 
+    	hotelNationalitySelect.click();
+    	hotelNationalitySelect.select("Argentina");
+    	
+    	hotelSearchButton.click();
+    }
+    
+    public void flightsSearchForm(String flyingFrom, String toDestination, String departureDate, String adults, String childs, String infants) {
+    	
+    	flightsRoundTripRadioButton.click();
+    	
+    	flightsFlyingFromCombobox.click();
+    	flightsFlyingFromCombobox.type(flyingFrom);
+    	flightsFlyingFromSearch.click();
+    	
+    	flightsFlyingToCombobox.click();
+    	flightsFlyingToCombobox.type(toDestination);
+    	flightsFlyingToSearch.click();
+    	
+    	flightsDepartureDate.click();
+    	flightsDepartureDate.type(departureDate);
+    	
+    	flightsTypeDropdown.click();
+    	flightsTypeDropdown.select("Business");
+    	
+    	flightsPassengersDropdown.click();
+    	flightsAdults.type(adults);
+    	flightsChilds.type(childs);
+    	flightsInfants.type(infants);
+    	
+    	flightsSearchFlightButton.click();
+    
+    	
+    }
```

Comment:
You have long methods. It's not bad in some cases. lets add here LOGGER.info to have logs. For example after entering data to Flying from field lets have
`LOGGER.info("Selected city " + flyingFrom + " in field Flying from")`